internal/pfcpctl/commands/sessions.go

```diff
@@ -18,6 +18,7 @@ type commonArgs struct {
 	GnBAddress string `short:"g" long:"gnb-addr" description:"The (g/e)nodeB address"`
 	SDFfilter string `short:"s" long:"sdf-filter" description:"The SDF Filter to use"`
 	QFI uint8 `short:"q" long:"qfi" description:"The QFI value for QERs. Max value 64."`
+	GateStatus bool `short:"t" long:"gate-closed" description:"If set, the application QER gate status will be CLOSED"`
```

Comment:
Can we change the name of the variable as well? I believe it would improve readability and clarify better the meaning of the value as well.

Reply:
This applies also to other places in the PR, where we use GateStatus to identify when the gate is closed

Reply:
Yep. Changed also the protobuf's field name to `gateClosed`